packages/textmate-grammars/src/browser/perl.ts

```diff
@@ -22,6 +22,32 @@ export class PerlContribution implements LanguageGrammarDefinitionContribution {
 
     readonly id = 'perl';
     readonly scopeName = 'source.perl';
+    readonly config: monaco.languages.LanguageConfiguration = {
+        "comments": {
```

Comment:
When writing JSON in TypeScript, you can forget about the double quotes.

See: https://github.com/theia-ide/theia/blob/master/packages/cpp/src/browser/cpp-grammar-contribution.ts#L24-L54

Reply:
Ty for the feedback, made some changes